src/main/scala/com/evolutiongaming/scache/LoadingCache.scala

```diff
@@ -229,10 +228,13 @@ object LoadingCache {
             entryRefs
               .values
               .toList
-              .foldMapM { _.release.start }
+              .map(_.release)
```

Comment:
There's no Monoid for Fiber in CE3, seems like this can be an alternative (or parTraverse)

Reply:
parFoldMapA { _.release.uncancellable }